app/src/main/res/layout/fragment_comment_list.xml

```diff
@@ -15,10 +15,16 @@
         sothree:umanoDragView="@id/commentDragView">
 
         <!-- MAIN CONTENT: COMMENT LIST -->
+
         <include
-            layout="@layout/post_list_layout"
+            android:id="@+id/commentRecyclerView"
```

Comment:
I think the code would be simpler if we don't use an id and reuse `post_list_layout `for all list fragments, without passing an id as an argument